test/e2e/policySet_e2e_test.go

```diff
@@ -34,6 +35,7 @@ var _ = Describe("Test policy set", func() {
 			By("Creating " + testPolicySetYaml + " result is " + output)
 			Expect(err).To(BeNil())
 
+			By("Checking " + testPolicyName + " exists on the hub in ns " + userNamespace)
```

Comment:
Please move these changes to separate commits/PRs and they are not related to PSP test case.